itou/prescribers/management/commands/update_refused_prescriber_organizations_kind.py

```diff
@@ -0,0 +1,60 @@
+from django.core.management.base import BaseCommand
+from django.db.models import Count
+
+from itou.prescribers.models import PrescriberOrganization
+
+
+class Command(BaseCommand):
+    """
+    The command updates "kind" value of PrescriberOrganizations with "refused" authorization status
+    """
+
+    help = "Update kind of prescriber_organization with REFUSED authorization status"
+
+    def show_counts(self, prescriber_orgs_refused):
+
+        self.stdout.write("Number of Prescriber Organizations with REFUSED authorization status :")
+
+        prescriber_orgs_count = prescriber_orgs_refused.values("kind").annotate(Count("kind"))
```

Comment:
```suggestion
        prescriber_orgs_list_with_count = prescriber_orgs_refused.values("kind").annotate(Count("kind"))
```

Pour bien signifier que ce qui est renvoyé est une liste.